samples/cpp/benchmark_app/main.cpp

```diff
@@ -277,21 +275,21 @@ int main(int argc, char* argv[]) {
             perf_counts = (device_config.at(ov::enable_profiling.name()).as<bool>()) ? true : perf_counts;
 
             // the rest are individual per-device settings (overriding the values set with perf modes)
-            auto setThroughputStreams = [&]() {
-                const std::string key = getDeviceTypeFromName(device) + "_THROUGHPUT_STREAMS";
+            auto setThroughputStreams = [&] {
                 if (device_nstreams.count(device)) {
                     // set to user defined value
                     auto supported_properties = core.get_property(device, ov::supported_properties);
-                    if (std::find(supported_properties.begin(), supported_properties.end(), key) ==
+                    if (std::find(supported_properties.begin(), supported_properties.end(), ov::num_streams) ==
```

Comment:
It seems it intersects with https://github.com/openvinotoolkit/openvino/pull/10149/ @sshlyapn 

If some plugin doesn't support new nstreams property, streaming is stop working for it.

Reply:
Also benchmark_app user can have stored json configuration files with old names. Maybe there is a sense to handle it somehow. Otherwise we can have a mix.

Reply:
The configuration API is not mapped as is on previous API. Some new properties has different value types and usage patterns. Each configuration value will need transition code in worst case. We dropped support of depricated configuration properties in tools with new API. So the same should be done for configuration files.